core/jsoncollector.class.inc.php

```diff
@@ -254,6 +254,9 @@ public function Prepare()
      */
     public function Fetch()
     {
+	    if (empty($this->aJson)) {
+		    return false;
+	    }
```

Comment:
I hope it will be correct this time..
```suggestion
		if (empty($this->aJson)) {
			return false;
		}
```

Reply:
Closing as this was solved globally
Thanks for your suggestion and efforts !